ioctl/util/util_test.go

```diff
@@ -64,3 +64,31 @@ func TestTrimHexPrefix(t *testing.T) {
 		require.Equal(expects[i], TrimHexPrefix(s))
 	}
 }
+
+func TestParseHdwPath(t *testing.T) {
+	r := require.New(t)
+
+	tests := []struct {
+		addressOrAlias string
+		a, b, c        uint32
+		err            string
+	}{
+		{"hdw::0/1/2", 0, 1, 2, ""},
+		{"hdw::0/1", 0, 1, 0, ""},
```

Comment:
great, this is a good comprehensive test suite

hdw::0/1 should return 0, 0, 1, when there are 2 values they should be treated as 0/b/c